flink-streaming-java/src/main/java/org/apache/flink/streaming/api/transformations/MultipleInputTransformation.java

```diff
@@ -40,4 +42,12 @@ public MultipleInputTransformation(
 		inputs.add(input);
 		return this;
 	}
+
+	@Override
+	public String getName() {
+		return String.format(
+			"%s [%s]",
```

Comment:
In the old chaining way with only one input, we have `source -> op1 -> op2` in the log and web UI.
If I get this formatting correctly, we would have `op1 [source1, source2] -> op2` now.
I'd naively assume that it's easier for users to read `[source1, source2] -> op1 -> op2`. WDYT? 
Ofc, it kinda assumes that chaining is printed with " -> " in JobGraphGenerator, which might be less ideal.

Also, how does it look for no source? Would it print `op []`? I guess we should have a special treatment for non-chained inputs.

Reply:
Good point!
I've changed it to `originalName` if there are no inputs, and to `[source1, source2]` otherwise.